mysite/test/factories/user.py

```diff
@@ -0,0 +1,11 @@
+import factory
+from django.contrib.auth.models import User
+
+
+class UserFactory(factory.django.DjangoModelFactory):
+    class Meta:
+        model = User
+        django_get_or_create = ('username', 'email')
+
+    username = 'john.doe'
```

Comment:
> Instead hard-coding the values for the factory, we can make use of fake-factory to generate random values each times. Refer the documentation for fake-factory
> The code might look like,
> 
> from faker import Factory as FakerFactory
> faker = FakerFactory.create()
> 
> class UserFactory(factory.django.DjangoModelFactory):
>     class Meta:
>         model = User
> 
>     username = factory.LazyAttribute(lambda x: faker.user_name())
>     email = factory.LazyAttribute(lambda x: faker.email())

Yes, then is great if I need to create 50 users w/ random names. At this point I need only one user and it is ok to use the same name. Also I do not want to add another one external dependancy now. May be for future complex test we can add this proposal.
BTW thanks for good comment!